media-libs/vidstab/vidstab-1.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+MY_PN="vid.stab"
+DESCRIPTION="Video stabilization library"
+HOMEPAGE="http://public.hronopik.de/vid.stab/"
+SRC_URI="https://github.com/georgmartius/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please use distfile name consistent with the directory inside it, i.e. `${MY_PN}-${PV}`. Also, please don't use `${MY_PN}` in github repo name — using full `https://github.com/georgmartius/vid.stab` will make it easier for people to copy-paste that into browser.

Both remarks considered, you can change `MY_PN` to `MY_P` and include version there.

Reply:
I am not 100% sure I got it right, please have a look shortly.